ember_debug/view-debug.js

```diff
@@ -2,7 +2,7 @@
 import PortMixin from "ember-debug/mixins/port-mixin";
 
 const Ember = window.Ember;
-const { guidFor, $, computed, run, Object: EmberObject, typeOf, Component, Controller, ViewUtils, A, LinkComponent, String } = Ember;
+const { guidFor, $, computed, run, Object: EmberObject, typeOf, Component, Controller, ViewUtils, A, LinkComponent, String, View } = Ember;
```

Comment:
This is for Ember >= 2.7, there's no `Ember.View`.  It's the [old code](https://github.com/emberjs/ember-inspector/blob/930fc378f232f85ab51c906a74556aab4ca17f46/ember_debug/view-debug.js#L555) that needs to be fixed (by removing `|| View.views`).